e2-studio-star-rx72n-firmware/src/shared/shared_data.c

```
@@ -1510,6 +1565,205 @@ rx_err_t shared_data_trigger_estop(estop_reason_t reason)
+ *
+ * @see shared_data_trigger_estop_isr_safe() ISR-safe e-stop trigger
+ * @see shared_data_trigger_estop() Task-context e-stop trigger
+ * @see shared_data_is_estop_active() Check if e-stop active
+ *
+ * @since Version 1.1.0
+ */
+rx_err_t shared_data_commit_isr_estop(void)
+{
+  UINT            tx_status;
+  UINT            saved_interrupt_state;
+  bool            pending;
```

Clang (14.0.6)

[warning] variable 'pending' is not initialized

(cppcoreguidelines-init-variables)